canonical/ubuntu.com
# [WD-4668] update kernel chart and improve charts in general

## Done

- Updated data for Kernel chart according to [this doc](https://docs.google.com/spreadsheets/d/1jeCbxjtvg6xGsYfOrTvhtTKc2LWx-sa3DrZ5B6McExI/edit#gid=0)
- Added title to Y axis ticks for Kernel chart and shortened its  Y ticks labels according to above mentioned doc
- Added horizontal tick lines to charts for better visibility
- Put legend at the top of the chart
- Removed unnecessary setTimeout from createChart function that was messing with rendering the chart

## QA

- Navigate to /about/release-cycle#ubuntu-kernel-release-cycle and check the Kernel chart
    - Check its data on small screens when it becomes a table
- Navigate to /kernel/lifecycle and check data on Kernel charts there too, it should have same changes as on /about-release-cycle#ubuntu-kernel-release-cycle except the "Coming soon" part
- Check all other charts at the following paths if they look good and don't have any UI/non-UI bugs:
    - /about
    - /about/release-cycle#ubuntu
    - /about/release-cycle#openstack-release-cycle
    - /about/release-cycle#canonical-kubernetes-release-cycle
    - /16-04
    - /server
    - /desktop/developers (doesn't have tick lines)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4668

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
